src/Hyperlink/index.jsx

```diff
@@ -48,6 +51,7 @@ Hyperlink.defaultProps = {
   onClick: () => {},
   externalLinkAlternativeText: 'Opens in a new window',
   externalLinkTitle: 'Opens in a new window',
+  isUnderline: false,
```

Comment:
Correct me if I am wrong, currently, all hyperlink compoents are underlined out of the box, so adding default prop to `false` would remove the underline and can change their behavior. What are your thoughts on this?

Reply:
I am not sure if the current ones are underlined I am short of context on this. I think someone from the paragon team can make this decision about what should be the default behavior of this.

Reply:
Please see my comment on this PR: https://github.com/edx/paragon/pull/739#issuecomment-856752410

We should implement links as they are reflected in the design system docs here: https://www.figma.com/file/eGmDp94FlqEr4iSqy1Uc1K/Paragon-2021?node-id=789%3A42 

We have the concept of "inline" vs "standalone" links. In our system inline links are underlined and standalone are not. This css https://github.com/edx/paragon/blob/master/scss/core/_typography.scss#L38 implements this behavior by default as best it can.